src/master-playlist-controller.js

```diff
@@ -72,7 +72,7 @@ export default class MasterPlaylistController extends videojs.EventTarget {
       withCredentials: this.withCredentials,
       seekable: () => this.seekable(),
       seeking: () => this.tech_.seeking(),
-      setCurrentTime: (a) => this.setCurrentTime(a),
```

Comment:
The tech is a big ball of shared state so I wish we could minimize the parts of the system that touch it.